### ISSUE TITLE
Problem with build: local.properties - no such file or directory

### ISSUE THREAD

**Issue by vbauer** [workarounds/typography#4]
Could not build project:

``` bash
➜  typography git:(master) ./gradlew assembleDebug

FAILURE: Build failed with an exception.

* Where:
Script 'https://gist.githubusercontent.com/madki/e85785036b9ef3b85c63/raw/1fd9bc500586ddd2780fe045aeea05c3bb72e0d8/bintrayv1.gradle' line: 26

* What went wrong:
A problem occurred evaluating script.
> /Users/vbauer/Documents/workspace/java/typography/local.properties (No such file or directory)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 7.937 secs
```

It will be useful to make the local.properties file an optional (it is needed for bintray publishing)

**Comment by okmanideep** [comment]
@vbauer looking into it

**Comment by okmanideep** [comment]
@vbauer doesn't Android Studio automatically generate local.properties?

**Comment by vbauer** [comment]
@manidesto I'm trying to build it from console without Android Studio. )

**Comment by madki** [comment]
@vbauer I'm not sure how building from console works, but even in android studio without local.properties it does not build. Studio generates this file and it's usually not checked into the version control. Please try creating a local.properties file in the root and add `sdk.dir=path/to/your/sdk`

**Comment by madki** [comment]
@vbauer if the problem is due to `bintrayv1.gradle` you could actually remove the `apply` lines at the bottom of `typography/build.gradle`. They have gradle tasks to upload to bintray, nothing to do with building the library.

**Comment by vbauer** [comment]
@manidesto Yes, I understand how to fix it locally.. I've just experimented with continuous integration server and tried to build this project without any changes in code (just "git clone") and without using Android Studio (so this suggestions will not help). Anyway, thank you for response!

**Comment by madki** [comment]
@vbauer Thanks for letting us know of the issue we'll figure out a better way.

**Comment by okmanideep** [comment]
@vbauer Fixed in b9b911b. Please check on your end.
